lib/componentRelay.ts

```diff
@@ -37,12 +36,27 @@ type ComponentData = {
   [key: string]: any
 }
 
+type ItemMessagePayload = {
```

Comment:
Can we import this from SNJS instead? We already have SNJS listed as a dependency, so might as well take advantage of it instead of duplicating code. You'd probably just have to export from SNJS.

Reply:
Yeah, that would be the ideal, but importing from SNJS increases the bundle size up to +300 KB, something that we may need to look into.

Reply:
You might be able to import just the types